### PR TITLE
feat(server): Add missing fields to INFO PERSISTENCE command (#1408)

### PR DESCRIPTION
**Pull request by kostasrim**
This PR adds `loading` and `rdb_changes_since_last_save` fields to `INFO PERSISTENCE` command. 
This PR addresses the issue #1408

### CODE REVIEW

File: src/server/server_family.cc [hunk 1]
```diff
@@ -1622,10 +1643,13 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
     append("last_save", save_info->save_time);
     append("last_save_duration_sec", save_info->duration_sec);
     append("last_save_file", save_info->file_name);
+    size_t is_loading = service_.GetGlobalState() == GlobalState::LOADING;
+    append("loading", is_loading);
```
Comment (kostasrim) [inline src/server/server_family.cc:1647]: Is there a way to test this in a simple way? I will need a large dataset to load and then during the loading phase I should issue an `INFO PERSISTENCE` command.

Comment (dranikpg) [reply src/server/server_family.cc:1647]: Never worked with the rdb unit tests, but we have pytests that load a fairly large snapshot

File: src/server/server_family.cc [hunk 2]
```diff
@@ -591,6 +591,20 @@ struct AggregateLoadResult {
...
+  OnScopeExit(const OnScopeExit&) = delete;
+  OnScopeExit(const OnScopeExit&&) = delete;
+  ~OnScopeExit() {
+    fun();
+  };
+
+ private:
+  T fun;
+};
+
```
[the remark this answers is not in the document]
Comment (kostasrim) [reply src/server/server_family.cc:607]: I thought I removed this, I didn't end up using it! +1 for `absl::Cleanup`. 

P.s. I thought only `folly` had this, but turns out `absl` does as well :D

File: src/server/server_family.cc (hunk 1 above)
Comment (kostasrim) [reply src/server/server_family.cc:1647]: Cheers, will take a loo

File: tests/dragonfly/snapshot_test.py [hunk 3]
```diff
@@ -191,3 +191,34 @@ async def test_snapshot(self, df_seeder_factory, df_server):
...
+    async def test_snapshot(self, df_seeder_factory, df_server):
+        seeder = df_seeder_factory.create(port=df_server.port, **SEEDER_ARGS)
+        await seeder.run(target_deviation=0.1)
+
+        df_server.stop()
+        df_server.start()
+
+        a_client = aioredis.Redis(port=df_server.port)
+        res = await a_client.execute_command("INFO PERSISTENCE")
+        assert ord('1') == self.extract_is_loading_field(res)
```
Comment (dranikpg) [inline tests/dragonfly/snapshot_test.py:217]: I had to think for a second what it actually means 😄 Can't you use the byte prefix like you did above, just extract not a char, but a subslice when you get the field

**Review by dranikpg — APPROVED** [review]

**Review by dranikpg — APPROVED** [review]

**Review by dranikpg — APPROVED** [review]

**Pull request merged**